test/unit/sql/where.test.js

```diff
@@ -908,6 +908,24 @@ suite(Support.getTestDialectTeaser('SQL'), () => {
       });
     }
 
+    if (current.dialect.supports.REGEXP) {
+      suite('$regexp', () => {
+        testsql('username', {
+          $regexp: '^sw.*r$'
```

Comment:
It looks like the tests for PostgreSQL are failing before they can get to the regexp code. I believe this is unrelated to my changes.